pkg/operation/botanist/component/vpnshoot/testdata/monitoring_alertingrules.yaml

```
@@ -0,0 +1,24 @@
+rule_files:
+- vpn.rules.yaml
+
+evaluation_interval: 30s
+
+tests:
+- interval: 30s
+  input_series:
+  # VPNShootNoPods
+  - series: 'kube_deployment_status_replicas_available{deployment="vpn-shoot"}'
+    values: '0+0x60'
+  alert_rule_test:
+  - eval_time: 30m
+    alertname: VPNShootNoPods
+    exp_alerts:
+    - exp_labels:
+        service: vpn
+        severity: critical
+        type: shoot
+        visibility: operator
+        deployment: vpn-shoot
+      exp_annotations:
+        description: vpn-shoot deployment in Shoot cluster has 0 available pods. VPN won't work.
+        summary: VPN Shoot deployment no pods
```

Nit: can you leave 1 empty line at the end of the file?

```suggestion
        summary: VPN Shoot deployment no pods

```